marpa/lib/Guacamole.pm

```diff
@@ -100,18 +111,279 @@ ArrowDerefCall    ::= CallArgs
 ArrowMethodCall   ::= Ident CallArgs
 ArrowIndirectCall ::= SigilScalar Ident CallArgs
 
+# All keywords
 OpKeyword ::= OpKeywordAbsExpr
```

Comment:
This rule does not seem to be used anywhere?

Reply:
I think this is also fixed now.